tempodb/backend/s3/s3_test.go

```diff
@@ -19,6 +19,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const TAGHEADER = "X-Amz-Tagging"
```

Comment:
nit:  Please rename these consts to be cased `tagHeader` and `storageClassHeader`